selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -367,6 +366,17 @@ private void captureScreenshot(ITestResult result, SeleniumWebDriver webDriver)
     }
   }
 
+  private void captureScreenshotsFromOpenedWindows(
+      ITestResult result, SeleniumWebDriver webDriver) {
+    webDriver
+        .getWindowHandles()
+        .forEach(
+            currentWin -> {
+              webDriver.switchTo().window(currentWin);
+              captureScreenshotFromWindow(result, webDriver);
```

Comment:
Is it possible to call `dumpHtmlCodeFromTheCurrentPage(result, webdriver)` from here to have html dumps for each screenshoting pages as well?

Reply:
We have not needed in this feature for html capturing. And I do not see the need for this for current state tests. But if will be the same precedent like for screenshots -we'll add this for html dumpimg.